src/docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -949,14 +949,15 @@ server.3=zoo3:2888:3888</programlisting>
           </varlistentry>
 
           <varlistentry>
-            <term>ttlNodesEnabled</term>
+            <term>zookeeper.extendedTypesEnabled</term>
```

Comment:
Do we are have any room left for this? iiuc ttls are the last.

My concern here - won't people be confused by this, e.g. "are containers extended types"?

What do you think @Randgalt ?

Reply:
I only thought that we might be able to use the setting for other things in the future. But, I'm OK either way. Let me know.